gnucash/gnome-utils/gnc-gnome-utils.c

```diff
@@ -421,7 +421,39 @@ gnc_gnome_help (const char *file_name, const char *anchor)
     gboolean success;
 
     if (anchor)
-        uri = g_strconcat ("ghelp:", file_name, "?", anchor, NULL);
+    {
+        const gchar * const *sdatadirs = g_get_system_data_dirs ();
+        const gchar * const *langs = g_get_language_names ();
+        gchar *lookfor = g_strconcat ("gnome/help/", file_name, NULL);
+        gchar *help_path = NULL;
+        gchar *help_file = NULL;
+        gchar *out = NULL;
+
+        for (; *sdatadirs; sdatadirs++)
+        {
+            gchar *filename = g_build_filename (*sdatadirs, lookfor, NULL);
+            if (g_file_test (filename, G_FILE_TEST_EXISTS))
+                help_path = g_strdup (filename);
+            g_free (filename);
+        }
+
+        help_file = g_strconcat (file_name, ".xml", NULL);
```

Comment:
".xml" should also get a more prominent place. The documenters could decide to use ".dokbook" or switch to another format.

Reply:
Yelp can handle .html, maybe we should switch to that (not in this PR of course). Even better, we could ditch yelp and Windows help viewer entirely and display the docs with gnc-html.